### PR TITLE
new grid test file

### PR DESCRIPTION
**Pull request by 9pool**

### CODE REVIEW

File: css-flex.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      font-weight: bold;
+
+    }
+    .flex1 {
+      display:flex;
+
+
+    }
+
+
```
Comment (bodytaing) [inline css-flex.html:30]: remove all the extra newlines when you commit

File: index2.html [hunk 2]
```diff
@@ -31,6 +31,11 @@
         margin: 0 auto;
         max-width: 1400px;
       }
+      [class*='col-'] {
+        background-color: #eee;
+        border: 1px solid red;
+      }
+
```
Comment (bodytaing) [inline index2.html:40]: Be sure to clean up all these extra newlines

File: index2.html [hunk 3]
```diff
@@ -101,32 +106,74 @@ <h2>Glauber Costa in ITNEXT</h2>
...
-      <h2>Main Content</h2>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas sit amet pretium urna. Vivamus venenatis velit nec neque ultricies, eget elementum magna tristique. Quisque vehicula, risus eget aliquam placerat, purus leo tincidunt eros, eget luctus quam orci in velit. Praesent scelerisque tortor sed accumsan convallis.</p>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas sit amet pretium urna. Vivamus venenatis velit nec neque ultricies, eget elementum magna tristique. Quisque vehicula, risus eget aliquam placerat, purus leo tincidunt eros, eget luctus quam orci in velit. Praesent scelerisque tortor sed accumsan convallis.</p>
-    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-8">
+      <div class="row">
+        <div class="col-8">
```
Comment (bodytaing) [inline index2.html:134]: Notice you have a duplicate, nested `col-8`, as well as a duplicated nested `row`

File: css-flex.html [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>FlexGridTest</title>
+  <style>
+    .flex-container {
+      display:flex;
+      flex: 0 1 auto;
+      flex-direction: row;
+
+
```
Comment (bodytaing) [inline css-flex.html:13]: Remove extra newlines

File: css-flex.html [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+      color: indigo;
+      font-weight: bold;
+
+    }
+    .flex1 {
+      display:flex;
+
+
+    }
+
```
Comment (bodytaing) [inline css-flex.html:29]: clean up the whitespace newlines

File: css-flex.html (hunk 1 above)
Comment (9pool) [reply css-flex.html:30]: Done

File: index2.html (hunk 2 above)
Comment (9pool) [reply index2.html:40]: Done

File: index2.html (hunk 3 above)
Comment (9pool) [reply index2.html:134]: Complete

File: css-flex.html (hunk 4 above)
Comment (9pool) [reply css-flex.html:13]: Done.

File: css-flex.html (hunk 5 above)
Comment (9pool) [reply css-flex.html:29]: Done.

**Comment by 9pool** [comment]
Changes made and merged branches for this PR.

**Pull request merged**